modules/types/src/channel.ts

```diff
@@ -35,7 +35,7 @@ export type CreateTransferParams = {
   assetId: string;
   transferDefinition: string;
   transferInitialState: TransferState;
-  timeout: string;
+  timeout?: string;
```

Comment:
Here and in the `SetupParams` we should enforce the presence. It's in the engine `schema` that this should happen, and these should be autofilled so that in the protocol it is required